app/src/lib/shell.ts

```diff
@@ -0,0 +1,155 @@
+/* tslint:disable:no-sync-functions */
+
+import * as ChildProcess from 'child_process'
+import * as os from 'os'
+
+const ENVIRONMENT_VARIABLES_TO_PRESERVE = new Set([ 'NODE_ENV', 'NODE_PATH' ])
```

Comment:
Sorry I didn't note this before. Why do we exclude these? Could we document why?